soc/ipl/main.c

```diff
@@ -409,12 +409,18 @@ usb_setup_serial_no(void)
 
 extern uint32_t *irq_stack_ptr;
 
+extern volatile uint32_t UART[];
+#define UARTREG(i) UART[i/4]
```

Comment:
Why's this added?

Reply:
Was for testing purposes.  I'm going to remove all my changes to ipl/main.c in my PR.  We should probably have a start-up chime, though.  I've got something simple in there now, and I've kinda grown to like it.  It's subtle, and only takes a few cycles off of the memory bus.

Reply:
We can possibly add it to the actual startup screens; that's a separate app.